internal/uvm/scsi/mount.go

```diff
@@ -44,6 +44,13 @@ type MountConfig struct {
 	Encrypted bool
 	Verity    *guestresource.DeviceVerityInfo
 	Options   []string
+	// EnsureFilesystem indicates if we should check that a mount has the
+	// target `Filesystem` FS formatted.
```

Comment:
To be specific, it means we will format to `Filesystem` if it isn't already formatted as that fs. Not just checking.

Reply:
Updated.